linux_os/guide/services/xwindows/disabling_xwindows/xwindows_remove_packages/anaconda/shared.anaconda

```diff
@@ -1,3 +1,3 @@
-# platform = multi_platform_all
+# platform = multi_platform_fedora,multi_platform_ol,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 9
```

Comment:
This is basically "not RHEL8 because of RHV" - please include this as a comment, so we can easily revert it when we dont' need this any more.